SMP supported doc (#3194)
harness/developer-hub: docs/feature-flags/smp-ff-roadmap.md

---

> title: Self Managed Platform Roadmap and feature availability

Reviewer: ```suggestion
title: Self-Managed Platform roadmap and feature availability
```

> description: The features available and the roadmap for the self-managed Enterprise edition of Harness Feature Flags.

Reviewer: ```suggestion
description: Lists the available features and the roadmap for the Self-Managed Enterprise Edition of Harness Feature Flags.
```

tags: 
   - feature flag
   - smp
   - self managed
sidebar_position: 200
helpdocs_is_private: false

> helpdocs_is_published: true

Reviewer: You don't need these attributes. They were included when we migrated content from the previous docs site, but are not needed for new topics.

---

# Self-Managed Enterprise Edition - Roadmap and feature availability

> The Harness Self-Managed Enterprise Edition of Feature Flags is a smart feature flags solution focused on helping you rollout changes with end to end governance, auditing and automation.

Reviewer: ```suggestion
The Harness Self-Managed Enterprise Edition of Feature Flags is a smart feature flags solution focused on helping you roll out changes with end-to-end governance, auditing, and automation.
```

> The table below outlines the roadmap for the Harness Self-Managed Enterprise Edition of Cloud Cost Management:

Reviewer: ```suggestion
The table below outlines the roadmap for the Harness Self-Managed Enterprise Edition of Feature Flags:
```

| **Release version**| **Feature set** | **Deployment infrastructure** | **Supported platforms** | **Supported ingress** |
| --- | --- | --- | --- | --- |

> | Beta | All functionalitys | <ul><li> AWS Commercial Cloud</li><li>Connected</li><li>Signed certificates</li></ul>| <ul><li>AWS Commercial and GovCloud</li><li>Kubernetes clusters</li></ul> | Istio virtual services in strict mode |

Reviewer: ```suggestion
| Beta | All functionality | <ul><li> AWS Commercial Cloud</li><li>Connected</li><li>Signed certificates</li></ul>| <ul><li>AWS Commercial and GovCloud</li><li>Kubernetes clusters</li></ul> | Istio virtual services in strict mode |
```

| Public Preview (Coming soon)| All functionality | <ul><li> AWS Commercial Cloud</li><li>Connected/Airgapped</li><li>Signed certificates</li></ul> | <ul><li>AWS Commercial and GovCloud</li><li>Azure commercial and GovCloud</li><li>Kubernetes clusters</li></ul> | Istio virtual services in strict mode |
| General Availability **(Current version)** | All functionality | 

> # SDK Support

Reviewer: ```suggestion
# SDK support in Self-Managed Enterprise Edition
```

> The Following SDKs **do currently support** Self-Managed Enterprise edition

Reviewer: ```suggestion
The Following SDKs **do currently support** Self-Managed Enterprise Edition:
```

- Go
- Java

> The Following SDKs **do _not_ currently support** Self-Managed Enterprise edition

Reviewer: ```suggestion
The Following SDKs **do _not_ currently support** Self-Managed Enterprise Edition:
```
